.github/workflows/lint-changed.yml

```diff
@@ -17,19 +17,41 @@ jobs:
     if: ${{ github.actor != 'OSBotify' || github.event_name == 'workflow_call' }}
     runs-on: ubuntu-latest
     steps:
+      - name: Fetch merge base SHA from API
+        id: merge_base
+        run: |
+          MERGE_BASE_SHA="$(gh api repos/"$REPO"/compare/"$PR_BASE_SHA"..."$PR_HEAD_SHA" \
+          | jq -r '.merge_base_commit.sha')"
+          echo "MERGE_BASE_SHA=$MERGE_BASE_SHA" >> "$GITHUB_OUTPUT"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO: ${{ github.repository	}}
+          PR_BASE_SHA: ${{ github.event.pull_request.base.sha }}
+          PR_HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+
+      - name: Count commits  between merge base and HEAD
+        id: count
+        run: |
+          COMMIT_COUNT="$(gh api repos/"$REPO"/compare/"$BASE"...main \
+            --jq '.total_commits')"
```

Comment:
Right?

```suggestion
            | jq -r '.total_commits')"
```

Reply:
Yeah `gh` cli has `jq` option built-in. We use it in a number of places 👍🏼